### DISCUSSION TITLE
PHP 8.2 support with openspout v4

### DISCUSSION THREAD

**Discussion by hmazter** [spatie/simple-excel#127] [Ideas]
Are there any plans to get this updated to use openspout v4, and with that support PHP 8.2?

I would happily give it a try for a PR.
But it seems that openspout v4 has some breaking changes, and would probably dictate that some methods change in this package as well. And I'm not sure of the direction this package should take 😄

**Comment by freekmurze** [comment]
We tagged v3.0 which brings support for PHP 8.2 and uses openspout v4 under the hood.

**Reply by hmazter** [reply]
Amazing!! 🎉
